### DISCUSSION TITLE
Upcoming v8.2.0.21 for stable & beta, for all browsers, with some minor fixes

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#82] [Announcements]
Give the update some time to land on the 3 different stores. Should be available in a few hours.

- fix some embedded vimeo videos: https://github.com/aclap-dev/video-downloadhelper/discussions/81
- fix in browser download: https://github.com/aclap-dev/video-downloadhelper/discussions/73
- fix audio only MPD: https://github.com/aclap-dev/video-downloadhelper/discussions/76
- fix default action binding: https://github.com/aclap-dev/vdhcoapp/issues/208

**Comment by intermezzzo** [comment]
Many Thanks!! Very fast. Amazing!
